pennylane/devices/tests/test_measurements.py

```diff
@@ -468,13 +517,13 @@ def test_hermitian(self, device, tol, skip_if):
         """Test that a tensor product involving qml.Hermitian works correctly"""
         n_wires = 3
         dev = device(n_wires)
-        skip_if(dev, {"tensor_observable": False})
+        skip_if(dev, {"supports_tensor_observables": False})
 
         theta = 0.432
         phi = 0.123
         varphi = -0.543
 
-        A_ = np.array(
+        A_ = 0.1 * np.array(
```

Comment:
This (and below) rescales the Hermitian to effectively rescale the result, so we can use ``atol``.

Reply:
Awesome!